src/Console/Output/Progress/FileInfoOutput.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Console\Output\Progress;
+
+use PhpCsFixer\FixerFileProcessedEvent;
+
+/**
+ * Ideas for improvements:
+ *   - clickable file name (open file in IDE, should have Docker support).
+ */
+abstract class FileInfoOutput extends OneCharLegendContextAwareOutput
+{
+    public function onFixerFileProcessed(FixerFileProcessedEvent $event): void
+    {
+        $status = parent::getStatusMap()[$event->getStatus()];
+        $this->getOutput()->writeln(sprintf(
+            '[%s] %s%s',
```

Comment:
why initial `[%s]` and not only `%s` ?

Reply:
Better separation from the path 🤷‍♂️. That's similar to how I commit Jira issue IDs 😉.